test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -2150,6 +2150,23 @@ public void SavePipeChooseColumnsByIndex()
             Done();
         }
 
+        [TestCategory("DataPipeSerialization")]
+        [Fact()]
+        public void SavePipeTextLoaderWithMoreOptions()
+        {
+            string dataPath = GetDataPath("multiline.csv");
+            const string loaderArgs = "loader=text{sep=, quote+ multilines+ header+ col=id:Num:0 col=description:TX:1 col=animal:TX:2}";
+
+            OutputPath modelPath = ModelPath();
+            string extraArgs = null;
+            TestCore("showdata", dataPath, loaderArgs, extraArgs);
+
+            _step++;
+
+            TestCore("showdata", dataPath, string.Format("in={{{0}}}", modelPath.Path), "");
+            Done();
+        }
```

Comment:
This test checks that the new options work from MAML command line. It tests both TextLoader and TextSave